File: model/src/main/java/com/indracompany/treinamento/model/dto/SaqueDTO.java
package com.indracompany.treinamento.model.dto;

import java.io.Serializable;

import lombok.Data;

@Data
public class SaqueDTO implements Serializable{

	private static final long serialVersionUID = -3353310391854167364L;
	
	private String agencia;
	private String numeroConta;
	private double valor;
	
}

File: view/src/app/core/dtos/saque.dto.ts
export class SaqueDTO {

    agencia: string;
    numeroConta: string;
    valor: number;

    constructor(obj?) {
        if (obj) {
            this.agencia = obj.agencia;
            this.numeroConta = obj.numeroConta;
            this.valor = obj.valor;
        }
    }
}
